sentry/sentry_whitebox_test.go

```diff
@@ -4,7 +4,6 @@ import (
 	"context"
```

Comment:
there's already a `sentry` pkg in https://github.com/fabric8-services/fabric8-common/tree/master/sentry, so what about removing this local pkg instead ?